### ISSUE TITLE
Unchecked access to document.activeElement in prose mirror-view/selection

### ISSUE THREAD

**Issue by cmlenz** [ProseMirror/prosemirror#1054]
### Issue details

We are seeing errors in Internet Explorer 11 with ProseMirror accessing `document.activeElement.contains` without checking whether `activeElement` is `null`.

The code in question is in [`selectionToDom`](https://github.com/ProseMirror/prosemirror-view/blob/6d542a421e5c40d6869e0931995024c042aa4740/src/selection.js#L34):

```js
  if (view.editable ? !view.hasFocus() : !(hasSelection(view) && document.activeElement.contains(view.dom))) return
```

Under normal circumstances it seems like IE11 will return the body when no particular element is focussed. But apparently it will return `null` when the previously focussed element got removed from the DOM.

I think it would be good to simply add a check for `null` here.

### Steps to reproduce

We haven't created a simple reproduction yet.

### ProseMirror version

prosemirror-view 1.14.5

### Affected platforms

- [ ] Chrome
- [ ] Firefox
- [x] Internet Explorer
- [ ] Other

### Screenshots / Screencast (Optional)

<img width="1088" alt="activeelementnull" src="https://user-images.githubusercontent.com/11300/81055467-0152dd80-8ec9-11ea-9ab8-a3b5197acf66.png">

**Comment by TheRealNic** [comment]
Thank you Marjin, this fixed our bug, could you please release a new version for this fix?

**Comment by marijnh** [comment]
This is released as 1.14.9 now
